specification/cognitiveservices/Language.AnalyzeText/models/pii.entity.recognition.tsp

```diff
@@ -32,12 +32,63 @@ model PiiTaskParameters is PreBuiltTaskParameters {
   stringIndexType?: StringIndexType = StringIndexType.TextElements_v8;
 
   /** Enumeration of PII categories to be excluded in the response. */
-  @added(Versions.v2023_11_15_preview)
+  @added(Versions.v2024_11_15_preview)
   excludePiiCategories?: Array<PiiCategoriesExclude>;
 
   /** Character to be used for redaction. Default in '*'. */
-  @added(Versions.v2023_11_15_preview)
+  @added(Versions.v2024_11_15_preview)
   redactionCharacter?: redactionCharacter = redactionCharacter.asterisk;
+
+  /** RedactionPolicy to be used on the input. */
+  @added(Versions.v2024_11_15_preview)
+  redactionPolicy?: BaseRedactionPolicy;
+}
+
+/** The abstract base class for RedactionPolicy. */
+@added(Versions.v2024_11_15_preview)
+@discriminator("policyKind")
+model BaseRedactionPolicy {
+  /** The entity RedactionPolicy object kind. */
+  policyKind: RedactionPolicyKind = RedactionPolicyKind.characterMask;
+}
+
+/** Represents the policy of redacting with a redaction character */
+@added(Versions.v2024_11_15_preview)
+model MaskWithRedactionCharacterPolicyType extends BaseRedactionPolicy {
+  /** The entity RedactionPolicy object kind. */
+  policyKind: RedactionPolicyKind.characterMask;
+
+  /** Optional parameter to use a Custom Character to be used for redaction in PII responses. Default character will be * as before. We allow specific ascii characters for redaction. */
+  redactionCharacter?: redactionCharacter = redactionCharacter.asterisk;
```

Comment:
do we need to mention that in typespec as redactionCharacter is an optional parameter

Reply:
Following the pattern of our optional parameters and keeping it for now.